### PR TITLE
Setup deployment to maven central

### PR DESCRIPTION
**Pull request by hzbarcea**
I think there are some plugin version upgrades we may want to take care before releasing.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,19 +49,15 @@
...
         <url>https://github.com/apifocal/apifocal-parent</url>
     </scm>
-    <ciManagement>
-        <system>Jenkins</system>
-        <url>https://jenkins.apifocal.org/job/Dev/job/apifocal-parent/</url>
-    </ciManagement>
 
     <distributionManagement>
         <repository>
-            <id>apifocal-releases</id>
```
Comment (CMoH) [inline pom.xml]: I think we should leave apifocal repository definitions towards archiva.apifocal.org and add ossrh. Would that be a problem?

Comment (hzbarcea) [reply pom.xml]: The only problem is that maven doesn't allow for multiple repositories in distributionManagement. I agree with the intention, but doesn't work.

Comment (CMoH) [reply pom.xml]: This branches our work into two sections: one that is using OSSRH and the other using archiva.apifocal.org. I guess OSSRH is supposed to be open-source, much like the name suggests.

Maybe have two parents? `apifocal-oss` for open-source and `apifocal-css` for closed-source software?

So far all projects rely on `apifocal-parent` and all docker images are built using archiva.apifocal.org. How do you envision to propagate this change? Reparent silkmq to `apifocal-css`, which is parented with `apifocal-parent` (the OSS one) ? Or have a common `apifocal-parent` with two child poms?

Comment (hzbarcea) [reply pom.xml]: Looks like there's a better way, I'll get it done.

**Pull request closed without merging**